### ISSUE TITLE
Missing controller errors

### ISSUE THREAD

**Issue by nijel** [phpmyadmin/error-reporting-server#80]
After upgrading to CakePHP 3.0 version logs show errors like:

```
2015-07-13 09:19:14 Error: [MissingControllerException] Controller class Robots.txtController could not be found.
Exception Attributes: array (
  'class' => 'Robots.txtController',
  'plugin' => NULL,
)
Request URL: /robots.txt?url=robots.txt
Stack Trace:
#0 /home/reports/error-reporting-server/app/webroot/index.php(108): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))
#1 {main}
```

CC @smita786

**Comment by devenbansod** [comment]
Hi @nijel, it seems that this has been fixed by #82 ?

**Comment by nijel** [comment]
Indeed, it seems to be fixed, thanks for spotting.
